dist/client.js

```diff
@@ -459,7 +459,6 @@
     define([], function() {
       return CrossStorageClient;
     });
-  } else {
-    root.CrossStorageClient = CrossStorageClient;
   }
+  root.CrossStorageClient = CrossStorageClient;
```

Comment:
Well, there are different ways to use Electron. In my case it's loaded dynamically via `<script>` and not pre-installed locally. 

I'm not arguing whether it's an issue with this library or not - it's doing what most libraries are doing for exports in different environments. I just thought it won't be a big sacrifice to export to global in order to support 1 more setup.